flint.ui/vue.config.js

```diff
@@ -1,4 +1,7 @@
 // vue.config.js
 module.exports = {
+  devServer: {
+    port: 8000
```

Comment:
Does this match our README instructions?

Reply:
The UI Docker container runs on port 8000. I prefer to run the local UI on 8000 too as that is the closest to hot reload. While the API containers run on 8080 (FLINT Example) & 8081 (GCBM), while debugging on 8000 (running locally, not on the UI container)is the most convenient as we can send requests to both containers without violating CORS policy.